modules/virt-restoring-vm-from-snapshot-cli.adoc

```diff
@@ -18,7 +18,7 @@ You can restore an existing virtual machine (VM) to a previous configuration by
 +
 [source,yaml]
 ----
-apiVersion: snapshot.kubevirt.io/v1beta1
+apiVersion: snapshot.kubevirt.io/v1alpha1
```

Comment:
**QE REVIEW NEEDED** Please verify the spacing in the YAML example.

Reply:
I think this needs QE review for the apiVersion change, not YAML spacing